client/src/ui/LandingNavBar.tsx

```diff
@@ -0,0 +1,144 @@
+"use client";
+import Image from "next/image";
+import Link from "next/link";
+import { Button, Typography } from "@mui/material";
+import { customTheme } from "@/theme/theme";
+import { ThemeProvider } from "@mui/material";
+
+
+/*
+ * This Landing Navbar will be responsible for rendering the Navbar of the Landing Page
+ */
+export function LandingNavbar() {
+  return (
+    <nav
+      style={{
+        background: "rgb(255, 255, 255)",
+        height: "80px",
+        width: "100%",
+        display: "grid",
+        zIndex: 999,
```

Comment:
nit: Can you not just do 1 for this value to render on top?

Reply:
It has been fixed!